charts/generic/k8-custom-metrics-apiserver/rbac-admin.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: v1
```

Comment:
@dharmaanu , this is a custom solution and don't have kubernetes resource associate, we don't need this chaos-chart.